src/Sylius/Behat/Context/Ui/Shop/ProductContext.php

```diff
@@ -445,6 +445,26 @@ public function iShouldSeeTheProductAssociationWithProducts($productAssociationN
     }
 
     /**
+     * @Then /^average rating of (product "[^"]+") should be (\d+)$/
+     */
+    public function thisProductAverageRatingShouldBe(ProductInterface $product, $averageRating)
```

Comment:
Yes, but this step was under private method, so I moved it before `assertIsProductIsInAssociation` and after that I added my new step. Therefore this lines are marks as new step.